src/components/CircularProgress/circularProgress.stories.tsx

```diff
@@ -8,8 +8,8 @@ import {
   Container
 } from '@chakra-ui/react'
 import { Story } from '@storybook/react'
-import { TestProviders } from 'jest/TestProviders'
```

Comment:
This was necessary because webpack interpreted the import as `node_modules/jest` instead of `src/jest`. Maybe I could get away without the relative path....